scripts/generate-deploy-yamls.sh

```diff
@@ -156,7 +168,7 @@ fi
 # update helm dependencies
 ( cd "$SCRIPTDIR"/../chart && helm dependency update )
 # generate the yaml
-helm template --set "$template_params" mayastor "$SCRIPTDIR/../chart" --output-dir="$tmpd" --namespace mayastor -f "$SCRIPTDIR/../chart/$profile/values.yaml"
+helm template --set "$template_params" mayastor "$SCRIPTDIR/../chart" --output-dir="$tmpd" --namespace mayastor -f "$SCRIPTDIR/../chart/$profile/values.yaml" --set "$helm_string" -f "$helm_file"
```

Comment:
Since most of the time the options will not be used we will be calling helm template with `--set "" -f ""` . However if helm does not complain then it is probably fine.

Reply:
Yehh, helm does not complain, just does nothing with that.